### PR TITLE
Rwa 54 complete task

### PR DESCRIPTION
**Pull request by chrisgrimble**
Added endpoint to task api to complete a task

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-54

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by lewis150193 — DISMISSED** [review]

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java
Comment (alexbaeza) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java:159]:
```suggestion
    public void should_return_a_204_when_completing_an_already_completed_task() {
```

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java
Comment (alexbaeza) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java:187]:
```suggestion

```
we can remove the pretty peek

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java
Comment (alexbaeza) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java:196]: this seems to work but perhaps we want to retrieve the task again using GET /task/{task-id}
and assert on the taskState however to do this it will need to wait until RWA-50 is merged as it contains the task mapping and stitching of the localvariables

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java
Comment (alexbaeza) [inline src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java:87]: Let's talk about this offline :)

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java
Comment (chrisgrimble) [reply src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java:196]: Think we talk about why I would rather do this this way.

**Review by alexbaeza — APPROVED** [review]

**Review by alexbaeza — APPROVED** [review]

**Pull request merged**
